src/quantum/azext_quantum/storage.py

```diff
@@ -0,0 +1,115 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
should we put this file under operations/ folder like others?

Reply:
My rationale for the `storage.py` file placement:
The `.py` files currently in `operations` are all named after `az quantum` commands, except for `__init__.py`.  The `.py` files at the next level up, in the `aztext_quantum` folder, contain more general utilitarian functions, so that’s why I put `storage.py` there – it’s not named after an `az quantum` command.

Reply:
Make sense. in this case should we rename file as _storage.py to align with other files that are not exposed as az command?

Reply:
Good idea!  That also highlights the difference between this repo's _storage.py and the qdk-python storage.py